Cargo.lock

```diff
@@ -967,6 +967,7 @@ dependencies = [
  "tracing-core",
  "tracing-subscriber",
  "uint",
+ "wagyu-zcash-parameters",
```

Comment:
I would like to block this PR on checking that `wagyu-zcash-parameters` results in the same parameters as loading the old parameter files. Perhaps we can check this automatically against the same hashes that were used in `fetch-params.sh`?

Reply:
If it helps, the `wagyu-zcash-parameters` crate also has a test you can invoke to check the parameters. Just do `cargo vet inspect wagyu-zcash-parameters 0.2.0 --mode=local` and copy the parameters to the right place (`mkdir src/archived && cp ~/.zcash-params/sapling-spend.params src/archived/sapling-spend.params && cp ~/.zcash-params/sapling-output.params src/archived/sapling-output.params`) and run `cargo test --release` to check.

Reply:
I did it a different way: I audited the entire code (excluding tests) of `wagyu-zcash-parameters*` as vendored (i.e. in `depends/<arch>/vendored-sources/wagyu-zcash-parameters*` after a build on this PR branch, which ensures I'm auditing code with the same hashes as in `Cargo.lock`), and observed that:
* it concatenates `wagyu-zcash-parameters-$N/src/sapling-spend-$N.params` for N = {1..5} to get `sapling-spend.params`;
* it reads `wagyu-zcash-parameters-6/src/sapling-output-1.params` to get `sapling-output.params`.

So I did this:
```shell
$ cat wagyu-zcash-parameters-1/src/sapling-spend-1.params wagyu-zcash-parameters-2/src/sapling-spend-2.params wagyu-zcash-parameters-3/src/sapling-spend-3.params wagyu-zcash-parameters-4/src/sapling-spend-4.params wagyu-zcash-parameters-5/src/sapling-spend-5.params >sapling-spend.params
$ diff -b sapling-spend.params ~/.zcash-params/sapling-spend.params 
$ diff -b wagyu-zcash-parameters-6/src/sapling-output-1.params ~/.zcash-params/sapling-output.params
```
No output from `diff -b` means the files are identical.